phpBB/styles/prosilver/theme/common.css

```diff
@@ -603,6 +604,53 @@ a.header-avatar span:after {
 	display: none !important;
 }
 
+/* Smiley Dropdown---------------------------------------- */
+.smiley-box {
+	float: right;
+	width: 100%;
+}
+
+.smiley-box .smiley-image {
+	margin: 3px;
+}
+
+.dropdown .smiley-content {
+	width: 200px;
+}
+
+.smiley-box .smiley-list-item {
+	border: none;
+	display: inline-block;
+}
+
+.smiley-list-item .smiley-add {
+	padding: 0;
+}
+
+.button .dropdown-left .dropdown {
+	right: -12px;
+}
+
+.button .dropdown-right .dropdown {
+	left: -12px;
+}
+
```

Comment:
To center the pointer. If it's not important I'll remove that. But it looks better when the pointer at the center.
![image](https://cloud.githubusercontent.com/assets/12277453/25318051/d0a45864-28a3-11e7-9d48-11b24c697e71.png)
![image](https://cloud.githubusercontent.com/assets/12277453/25318061/0829579e-28a4-11e7-81ef-ed2a3d1b8730.png)